Delete_Earn.py

```diff
@@ -0,0 +1,32 @@
+#// Time Complexity : O(maximum element in the given array nums)
+#// Space Complexity : O(maximum element in the given array nums)
+#// Did this code successfully run on Leetcode : Yes
+#// Any problem you faced while coding this : No
+
+
+#// Your code here along with comments explaining your approach
+#In this problem first we take the maximum element in the list and create a new array with that size and fill the newly created array at #the index where element value is equal to index and keep on adding the value to that index till we are donw with all the elements.with #the newly created array we can use the same logic used for the problem House_robber.we get the maximum number of points by taking the #maximum of the last row.
+
+class Solution(object):
+    def deleteAndEarn(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        if nums == None or len(nums) == 0:
+            return 0
+        
+        out=[0]*(max(nums)+1)
+        
```

Comment:
Don't do this for matrixes. For array `[0]*(max(nums)+1)`is fine. Remind me to cover it in class one more time. We had this discussion in 2nd class.